CMakeLists.txt

```diff
@@ -221,7 +221,8 @@ set(CMAKE_EXE_LINKER_FLAGS \"-lstdc++ -lgcc -lrt\" CACHE STRING \"Initial cache\
           execute_process(COMMAND echo __GLIBC_MINOR__ COMMAND "${CMAKE_CXX_COMPILER}" -E -P -imacros stdlib.h - OUTPUT_VARIABLE GLIBC_MINOR_)
           string(STRIP "${GLIBC_MINOR_}" GLIBC_MINOR)
           if("${GLIBC_MAJOR}.${GLIBC_MINOR}" VERSION_LESS "2.12")
-            set(UseCustomLibUV ON)
+            # set(UseCustomLibUV ON)
```

Comment:
The `UseCustomLibUV` code path is still there. It could probably be cleaned up ?

Reply:
now cleaned up